ONSdigital/sdc-responses-dashboard
# Add home page for dashboard

### What is the context of this PR?
Added the ability to choose a survey and collection exercise from a home page.
Don't have to painfully work out collex anymore :)

### How to review 
1. Ensure you are able to go to the homepage `/dashboard`.
2. Ensure you able to select and view provided surveys.
3. Ensure links to homepage work. (Clicking name in top left corner)
4. Modal is not toggleable on homepage.
5. App works as expected.
